databases/mysql8/Portfile

```diff
@@ -66,7 +65,8 @@ if {$subport eq $name} {
     use_parallel_build  yes
 
     # Requires c++14 (gcc 6.1 or newer, clang-3.4 or newer support c++14)
-    compiler.blacklist-append {*gcc-[3-4].*} *gcc-5 {clang < 900} macports-clang-3.3
+    compiler.cxx_standard 2014
+    compiler.blacklist-append *gcc-5 {clang < 900}
```

Comment:
The replacement of PortGroup cxx11 1.1 with compiler.cxx_standard 2014 is obviously correct.

The comment stating that it requires C++14 can be removed since that's now made obvious through the compiler.cxx_standard line.

According to the compiler selection wiki page, gcc 5 and newer support C++14, so blacklisting gcc-5 should not be necessary. If gcc-5 doesn't work, the comment should better explain what the problem is.

It would also be good if the port explained why it is blacklisting {clang < 900}.

Reply:
I am gathering errors to open tickets for, and can then at least leave links to these in the portfile.

So far with gcc5 I observe a problem configuring in trace mode, and an error when building without trace mode. I wonder if whoever excluded gcc5 might be able to provide a better explanation (@kencu, @cjones051073: do you recall why?).

I notice that [upstream documentation](https://dev.mysql.com/doc/refman/8.0/en/source-installation-prerequisites.html) only mentions Xcode 9 or later for building on macOS, but the port bypasses this check for this (see [trac#58579](https://trac.macports.org/ticket/58579)).

Reply:
See [trac#60893](https://trac.macports.org/ticket/60893) and [trac#60894](https://trac.macports.org/ticket/60894)

Reply:
I believe I "fixed" the gcc-5 blacklist to work, but didn't initiate that blacklist in the first place.

Good to look this over now and then -- things certainly change as versions come out, and we don't often enough go back and retest. Can take time, though.

Reply:
I added a comment linking to the ticket I opened regarding gcc5 (though maybe I am not seeing the original issue), and am forgoing clearly documenting why Xcode < 9.0 is blacklisted (all I find is that `{clang < 800}` appeared in 935f008618, then was replaced with `{clang < 900}` in 19e1a68971).

Reply:
Never mind, I am not adding links to any tickets. I am only going to remove the compilers obviously excluded by `compiler.cxx_standard 2014`. Explaining `compiler.blacklist-append *gcc-5 {clang < 900}` will have to be done by someone else who knows what they're doing. I thought `*gcc-5` might also instead be clarified as `macports-gcc-5` (as done by far more portfiles), but I will leave it alone too.